capstone-backend/src/main/java/com/google/sps/data/BookList.java

```diff
@@ -17,77 +17,89 @@
  */
 public final class BookList {
 
-  private Long id;
-  private Long userID;
-  private String bookshelfID;
-  private Collection<Long> collaboratorsIDs;
+  private String id;
+  private String userID;
+  private Collection<String> gbookIDs;
+  private Collection<String> collaboratorsIDs;
 
-  public BookList(Long userID, String bookshelfID, Collection<Long> collaboratorsIDs, Long id) {
```

Comment:
We also need to include a default empty constructor in order to use this in firestore. See: https://firebase.google.com/docs/firestore/manage-data/add-data#custom_objects.

Reply:
Edit: Java comes with default constructor, but if you want to write your own you can do that as well.

Reply:
eg. passing empty Strings instead of null (null would be default).